assets/src/edit-story/constants.js

```diff
@@ -24,8 +24,12 @@ export const PAGE_NAV_PADDING = 60;
 export const PAGE_NAV_BUTTON_WIDTH = 40;
 export const PAGE_NAV_WIDTH = PAGE_NAV_PADDING + PAGE_NAV_BUTTON_WIDTH;
 
+//QQQQQ: also need to update all onInsert
+// @todo: Common 9:16 resolutions are 900:1600, 1080:1920.
 export const PAGE_WIDTH = 412;
 export const PAGE_HEIGHT = 732;
+export const DEFAULT_EDITOR_PAGE_HEIGHT = 412;
+export const DEFAULT_EDITOR_PAGE_WIDTH = 732;
```

Comment:
Shouldn't the numbers for `DEFAULT_EDITOR_PAGE_WIDTH` and `DEFAULT_EDITOR_PAGE_HEIGHT` be swapped here? 🤔 

In which case they would be the same as `PAGE_WIDTH` and `PAGE_HEIGHT`

Reply:
Oops

Reply:
Fixed. It went unnoticed, b/c reversing these essentially disabled default size for the responsive case.

Reply:
@dvoytenko Can you push your fix (after a `git pull --rebase`)?

Reply:
Yes. Going right now.